### PR TITLE
[BUG] PMP-2034 | JAN-831

### PR DESCRIPTION
**Pull request by dtiwarATS**

### CODE REVIEW

File: assets/src/components/parts/janus-mcq/MultipleChoiceQuestion.tsx [hunk 1]
```diff
@@ -828,7 +828,11 @@ const MultipleChoiceQuestion: React.FC<PartComponentProps<McqModel>> = (props) =
   }
 
   return ready ? (
-    <div data-janus-type={tagName} style={styles} className={`mcq-input`}>
+    <div
+      data-janus-type={tagName}
+      style={styles}
+      className={`mcq-input ${customCssClass === 'no-wrap' ? customCssClass : ''}`}
```
Comment (dtiwarATS) [inline assets/src/components/parts/janus-mcq/MultipleChoiceQuestion.tsx:834]: Need to add custom class if it is 'no-wrap'. Rest of the classes already gets applied on custom component. I am not sure if we can add 'no-wrap' class from override file. @bsparks  thoughts?

**Comment by dtiwarATS** [comment]
closing this PR. Something has happened and these changes are not resolving the issues now. They were working 2 days back. I am not sure if something changed somewhere that is having impact on this.

**Comment by dtiwarATS** [comment]
@bsparks , I saw a strange behavior. The changes in this PR work when I point the SIM URL to

https://vft.inspark.education/VFTDresser/panos/DresserTTG4/Dresser.html

but it does not work with

https://vft.asu.edu/migrated/VFTDresser/panos/DresserTTG4/Dresser.html

It seems to be some issue with this VFT URL.

File: assets/src/components/parts/janus-mcq/MultipleChoiceQuestion.tsx (hunk 1 above)
Comment (bsparks) [reply assets/src/components/parts/janus-mcq/MultipleChoiceQuestion.tsx:834]: is it not possible to adjust via css using something like `janus-mcq.no-wrap .mcq-input {`

Comment (dtiwarATS) [reply assets/src/components/parts/janus-mcq/MultipleChoiceQuestion.tsx]: Done. I will update the CSS override file

**Review by bsparks — APPROVED** [review]

**Pull request merged**